example-library/src/main/java/uk/gov/gchq/palisade/example/library/rule/EmployeePassThroughRule.java

```diff
@@ -27,4 +27,9 @@
     public Employee apply(final Employee record, final User user, final Context context) {
         return record;
     }
+
+    @Override
+    public boolean isApplicable(final User user, final Context context) {
```

Comment:
Just wondering if this would make more sense the other way around isPassthrough?

Reply:
The method belongs a a class named Rule.  In that context "is applicable" is self explanatory whereas "is pass through" needs an explanation.  For that reason, I would argue we keep this as is.